contracts/l1-l2-messaging/run_e2e.sh

```diff
@@ -20,28 +23,30 @@ pushd ./solidity
 forge script ./script/L1L2.s.sol:Deploy --broadcast --rpc-url $ETH_RPC_URL
 popd
 
+MAX_FEE=99999999999999999999
+
 # Declare and deploy l1-l2 contract on L2.
-starkli declare ./cairo/target/dev/cairo_l1_l2.contract_class.json
-starkli deploy 0x0211fd0483be230ba40d43f51bd18ae239b913f529f95ce10253e514175efb3e --salt 123
+starkli declare ./cairo/target/dev/cairo_l1_l2.contract_class.json --max-fee-raw $MAX_FEE
+starkli deploy 0x0211fd0483be230ba40d43f51bd18ae239b913f529f95ce10253e514175efb3e --salt 123 --max-fee-raw $MAX_FEE
```

Comment:
should we extract also `0x0211fd0483be230ba40d43f51bd18ae239b913f529f95ce10253e514175efb3e` to a constant together with others?

Reply:
Added comment in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/299/commits/22f4ec9be263daabb71c6dc8cad998b2e0b85c7c